db/migrate/20180117025350_add_multiple_file_submission_to_question_programming.rb

```diff
@@ -3,9 +3,14 @@ def up
     add_column :course_assessment_question_programming, :multiple_file_submission, :boolean, default: false, null: false
 
     Course::Assessment::Question::Programming.find_each do |question|
-      service = Course::Assessment::Question::Programming::ProgrammingPackageService.new(question, nil)
-      meta = service.extract_meta
-      question.update_attribute(:multiple_file_submission, true) if meta && meta[:data] && meta[:data]['submit_as_file']
+      if question.language.is_a?(Coursemology::Polyglot::Language::Java)
```

Comment:
Have this in the scope above, to avoid initialising the question as a ruby object to check whether it is a Java question. So something like: 
```ruby
Course::Assessment::Question::Programming.where(xxx).find_each do |question|
  do_your_stuff_here
end
```

Reply:
OK fixed